README.rst

```diff
@@ -32,19 +32,24 @@ Getting Started
    defined on this `wiki page <https://openedx.atlassian.net/wiki/display/AN/Learner+Analytics#LearnerAnalytics-ElasticSearch>`_.
    The connection to elasticsearch can be configured by the
    ``ELASTICSEARCH_LEARNERS_HOST`` and
-   ``ELASTICSEARCH_LEARNERS_INDEX`` django settings.  For testing, you
-   can install elasticsearch locally:
+   ``ELASTICSEARCH_LEARNERS_INDEX`` django settings. To run tests locally use:
 
    ::
 
-      $ make test.install_elasticsearch
+      $ make test
 
-   Elasticsearch requires `JDK 1.8`_; install that if you don't have it or have a different version. To run the cluster for testing:
+   For testing is used docker Elasticsearch; to run ES container:
```

Comment:
`make test` will call `make test.run_elasticsearch` right? Could you rephrase it to something like: "This will launch an Elasticsearch Docker container for the tests. To simply run the container without starting the tests, run:"? Other than that, it looks like it's good to go

Reply:
thanks)